apps/condo/domains/common/components/tasks/TaskProgress.tsx

```diff
@@ -141,12 +141,23 @@ export const TaskProgressTracker: React.FC<ITaskProgressTrackerProps> = ({ task
         }
     })
 
-    const handleRemoveTask = useCallback(() => {
+    const removeTaskFromUI = () => {
```

Comment:
It's better to use `useCallback` with `deps` containing `deleteTaskFromContext` and `removeTaskFromStorage`